Create AGP-CS: Develop Aragon Chain (#106)
aragon/AGPs: AGPs/AGP-CS.md

---
AGP: N/A
Title: Develop and launch Aragon Chain
Author: Aidan Hyman & Gregory Markou
Status: Stage III
Track: Finance
Created: 2019-10-03
---

# Develop and launch Aragon Chain

## Address of the transfer recipient

> TBD

Reviewer: I recommend that this section be completed with a recipient address before the Stage V deadline.

## Amount of the transfer

> Upon community review, a better estimate can be made based on the needs and requirements from the community.

Reviewer: This section will need to be completed with a final amount before the Stage V deadline.

> A total of $x, will be allocated to the Aragon Chain. Invoices will be sent to the Aragon Association, and compensation will be at their discretion.

Reviewer 2: Please don't forget to change `x` for the final amount before the proposal needs to be frozen

## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)

1

## Purpose of the transfer

At the moment, Aragon fully depends on Ethereum mainnet in order to function.

However, Ethereum may not be the best platform for all Aragon use cases. Some Aragon organizations may not want to pay the security and composability premium for being on Ethereum.

On top of this, we are seeing increased development and user experience costs [resulting from changes in the Ethereum protocol that break Aragon organizations](https://www.coindesk.com/ethereums-istanbul-upgrade-will-break-680-smart-contracts-on-aragon).

> Building a sovereign layer 1 protocol owned and operated by the Aragon community will ensure that we can keep enhancing Aragon, and even introduce protocol changes that make running decentralized organizations easier and cheaper. This way we can offer Aragon organizations another option parallel to Ethereum, with different security, cost and performance trade-offs. The idea is for this chain to be heavily focused and optimized for the Aragon use case, so other heavyweight use cases that are currently clogging the Ethereum network wouldn't necessarily have any incentives to run on the Aragon Chain.

Reviewer: ```suggestion
Building a sovereign layer 1 protocol owned and operated by the Aragon community will ensure that we can continue enhancing Aragon, and even introduce protocol changes that make running decentralized organizations easier and cheaper. This way we can offer Aragon organizations another option parallel to Ethereum, with different security, cost, and performance trade-offs. The idea is for this chain to be heavily focused and optimized for the Aragon use case, so other heavyweight use cases that are currently clogging the Ethereum network wouldn't necessarily have any incentives to run on the Aragon Chain.
```

> The Aragon Chain will be an EVM compatible Proof of Stake blockchain whose native staking token is derived from ANT (in a similar fashion as the Aragon Court's ANJ).

Reviewer: ```suggestion
The Aragon Chain will be an EVM-compatible Proof of Stake blockchain whose native staking token is derived from ANT (in a similar fashion as the Aragon Court's ANJ).
```

Aragon One has worked with ChainSafe Systems on a feasibility study on what the most fitting technology for building the Aragon Chain is. We are now certain of the way forward and it's ready to get into the specification and development phase.

> Feasibility report: TBD (to be made public ahead of the AGP phase V transition)

Reviewer: Noting the need to fill in the blank for this link

> The report goes indepth between the current leading staking chains: Polkadot and Cosmos. Throughout the report comparisons are made based on Aragon specific optimizations, and cross-chain communication.

Reviewer: ```suggestion
The report goes in-depth between the current leading staking chains: Polkadot and Cosmos. Throughout the report comparisons are made based on Aragon-specific optimizations, and cross-chain communication.
```

The Aragon Chain will be an EVM compatible Proof of Stake blockchain whose native staking token is derived from ANT (in a similar fashion as the Aragon Court's ANJ).

Aragon One has worked with ChainSafe Systems on a feasibility study on what the most fitting technology for building the Aragon Chain is. We are now certain of the way forward and it's ready to get into the specification and development phase.

Feasibility report: TBD (to be made public ahead of the AGP phase V transition)

The report goes indepth between the current leading staking chains: Polkadot and Cosmos. Throughout the report comparisons are made based on Aragon specific optimizations, and cross-chain communication.

Deliverables from this project:

- Publish Aragon Chain spec
- Develop full-node software that follows the Aragon Chain spec
- Launch Aragon Chain testnet
- Launch Aragon Chain mainnet

## Recipient information

> TBD

Reviewer 2: ```suggestion
```
